src/components/reportAFind/reportAFind.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import {
+  Typography,
+  List,
+  ListItem,
+  Checkbox,
+  FormGroup,
+  FormControl,
+  FormControlLabel,
+  Grid,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { Items } from "./FindData";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    maxWidth: 900,
+  },
+  list: {
+    backgroundColor: theme.palette.background.default,
+  },
+  title: {
+    margin: theme.spacing(4, 0, 2),
+    variant: "h1"
+  },
+  hr:{
+    border: "none",
+    background: "rgba(128,128,128, 0.5)",
+    height: "1px"
+  },
+  title1: {
+    margin: theme.spacing(8, 0, 2, 0),
+    textAlign: "justify",
+  },
+  main: {
+    margin: "10px 20px",
+  },
+  listItem: {
+    padding: theme.spacing(0, 2, 2, 0),
+  },
+
+  listText: {
+    letterSpacing: 10,
+    padding: theme.spacing(0, 0, 0, 1),
+    "& :hover": {
+      backgroundColor: "#ecf9f1",
+      padding: theme.spacing(0.5),
+      cursor: "pointer",
+    },
+  },
+}));
+
+const ReportAFind = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.main}>
+      <div>
+      </div>
+      <Grid item xs={12} md={6}>
+        <Typography className={classes.title}>
+            Products (Choose at least 1) 
+            <br/><hr className={classes.hr}/>
```

Comment:
using `<Divider />` component here will be cool

Reply:
I'll work on this and make a new PR